CONFIGURE.md

```diff
@@ -1,160 +1,243 @@
-=> #[ ... ]
->> user.activate!
-=> true
->> quit
-```
+### Managing Users via Rails Console
 
-### Giving Administrator/Moderator Permissions
+1. **Enter the Rails console:**
+   ```bash
+   $ bundle exec rails console
+   ```
 
-To give administrator or moderator permissions:
+2. **Find the user:**
+   ```ruby
+   >> user = User.find_by(:display_name => "My New User Name")
+   => #[ ... ]
+   ```
 
-```
-$ bundle exec rails console
->> user = User.find_by(:display_name => "My New User Name")
-=> #[ ... ]
->> user.roles.create(:role => "administrator", :granter_id => user.id)
-=> #[ ... ]
->> user.roles.create(:role => "moderator", :granter_id => user.id)
-=> #[ ... ]
->> quit
-```
+3. **Modify the user as desired:**
 
-## OAuth Consumer Keys
+   **Activate/confirm the user:**
+   ```ruby
+   >> user.activate!
+   => true
+   ```
 
-There are two built-in applications which communicate via the API, and therefore need to be registered as OAuth 2 applications. These are:
+   **Grant moderator role:**
+   ```ruby
+   >> user.roles.create(:role => "moderator", :granter_id => user.id)
+   => #[ ... ]
+   ```
 
-* iD
-* The website itself (for the Notes functionality)
+   **Grant administrator role:**
+   ```ruby
+   >> user.roles.create(:role => "administrator", :granter_id => user.id)
+   => #[ ... ]
+   ```
 
-You can register them by running the following rake task:
+4. **Exit the Rails console:**
+   ```ruby
+   >> quit
+   ```
 
-```
-bundle exec rails oauth:register_apps["My New User Name"]
-```
+## OAuth Setup
 
-This task registers the applications with the "My New User Name" user as the owner
-and saves their keys to `config/settings.local.yml`. When logged in, the owner should be able to see the apps on the [OAuth 2 applications](http://127.0.0.1:3000/oauth2/applications) page.
-
-Alternatively you can register the applications manually, as described in the next section.
-
-### Manually registering the build-in OAuth applications
-
-For iD, do the following:
-
-* Go to "[OAuth 2 applications](http://localhost:3000/oauth2/applications)" on the My settings page.
-* Click on "Register new application".
-* Unless you have set up alternatives, use Name: "Local iD" and Redirect URIs: "http://localhost:3000"
-* Check boxes for the following Permissions
-  * 'Read user preferences'
-  * 'Modify user preferences'
-  * 'Modify the map'
-  * 'Read private GPS traces'
-  * 'Upload GPS traces'
-  * 'Modify notes'
-* On the next page, copy the "Client ID"
-* Edit config/settings.local.yml in your rails tree
-* Add the "id_application" configuration with the "Client ID" as the value
-* Restart your rails server
-
-An example excerpt from settings.local.yml:
-
-```yaml
-# Default editor
-default_editor: "id"
-# OAuth 2 Client ID for iD
-id_application: "Snv…OA0"
-```
+There are two built-in applications which communicate via the API, and therefore need to be registered as OAuth 2 applications. These are:
```

Comment:
The new documentation for @AntonKhorev's rake task for setting up the oauth keys needs to be preserved here...